src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public abstract class Figure implements IDrawable {
+    protected Color color;
+
+    public Figure(Color color) {
+        this.color = color;
+    }
+
+    public abstract double getArea();
```

Comment:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/61669d1e-8069-49f1-bbb6-a99beda59c72)

Reply:
Thanks for help!